mava/configs/system/sac/ff_isac.yaml

```
@@ -0,0 +1,33 @@
+# --- Defaults FF-ISAC ---
+seed: 1
+n_envs: 16 # number of environments to run in parallel (vmapped)
+
+# --- Agent observations ---
+add_agent_id: False
+
+# --- RL hyperparameters ---
+# step related
+total_timesteps: 3e8
+explore_steps: 5000  # number of steps to take with random actions at the start of training
+update_batch_size: 1 # number of vectorised gradient updates per device.
+
+rollout_length: 1 # number of environment steps per vectorised environment.
+epochs: 1 # number of learn epochs per training data batch.
+policy_frequency: 4  # the frequency of training policy - every n q learning steps the policy is trained once
```

I don't quite follow this?

---

It's a DDPG thing, but useful in SAC also, you delay policy updates. So you'll do 4 Q updates and then you update the policy 4 times

---

Ah makes sense! Can we then call it something along these lines please? :pray: 
```suggestion
policy_update_frequency: 4  # the frequency of training policy - every `n` q network learning steps the policy network is updated once
```

---

Did it slightly differently, but similar